src/libgit2/checkout.c

```diff
@@ -2578,7 +2586,6 @@ int git_checkout_iterator(
 		GIT_DIFF_INCLUDE_TYPECHANGE_TREES |
 		GIT_DIFF_SKIP_BINARY_CHECK |
 		GIT_DIFF_INCLUDE_CASECHANGE;
-	diff_opts.skip_sparse_files = 1;
```

Comment:
Is `skip_sparse_files` an option we added that we can delete?

Reply:
I'll remove it as we're no longer using it following this PR.